templates.html

```diff
@@ -0,0 +1,203 @@
+<div class="qa-templates" style="display: none">
+
+    <div class="js-page-hdr">
+        <div class="qa-tbl-row qa-tbl-hdr">
+            <div class="qa-page-id qa-center">
+                <span>Page ID</span>
+            </div>
+            <div class="qa-page-title">
+                <span>Page Title</span>
+            </div>
+            <div class="qa-page-q-count qa-center">
+                <span>Question Count</span>
+            </div>
+            <div class="qa-page-preview qa-center">
+                <span>Preview</span>
+            </div>
+        </div>
+    </div>
+
+    <div class="js-page-row">
+        <div class="qa-tbl-row page js-qa-page-row">
+            <div class="qa-page-id qa-center">
+                <span></span>
+            </div>
+            <div class="qa-page-title">
+                <span></span>
+            </div>
+            <div class="qa-page-q-count qa-center">
+                <span></span>
+            </div>
+            <div class="qa-page-preview qa-center">
+                <button type="button" class="btn btn-success page-preview">
```

Comment:
A tl;dr for choosing between a button and an anchor element:

- button for doing something
- anchor for going somewhere

So, in this case the correct markup is:

```html
<a href="#link_to_page" class="btn btn-success page-preview" target="_blank" rel="noopener">
   <span class="glyphicon glyphicon-eye-open"></span>
</a>
```